bgc_data/bgc_chemistry.dataresource.yaml

```diff
@@ -0,0 +1,117 @@
+profile: tabular-data-resource
+name: bgc_chemistry
+path: https://www.cmar.csiro.au/geoserver/imos/wfs?service=wfs&version=1.1.0&request=GetFeature&outputformat=csv&typename=imos%3Abgc_chemistry
+title: bgc_chemistry
+description: A table of fixed station information
+format: csv
+mediatype: text/csv
+encoding: utf-8
+schema:
+  fields:
```

Comment:
The order of the fields in this schema doesn't match the table I currently get from the WFS. And again, the column names are all uppercase.

Reply:
bgc_chemistry.dataresource.yaml removed for now (to be added again later)